### PR TITLE
Adding emails_input wdiget (will be used as part of the new invite process)

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Add a emails input widget, this will be used as part of the new invites
form which will be included in a future PR.

#### Ticket Link
[MM-14769](https://mattermost.atlassian.net/browse/MM-14769)

### CODE REVIEW

File: components/widgets/inputs/emails_input.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        value: PropTypes.arrayOf(PropTypes.string),
+    }
+
+    getCreateLabel = (value) => (
+        <React.Fragment>
+            <MailPlusIcon className='mail-plus-icon'/>
+            <FormattedMarkdownMessage
+                key='widgets.emails_input.valid_email'
+                id='widgets.emails_input.valid_email'
+                defaultMessage='Invite **{email}** as a channel guest'
```
Comment (grundleborg) [inline components/widgets/inputs/emails_input.jsx]: This text seems very specific for a component called `EmailsInput`.

File: components/widgets/inputs/emails_input.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                values={{email: value}}
+            />
+        </React.Fragment>
+    );
+
+    NoOptionsMessage = (props) => (
+        <FormattedMessage
+            id='widgets.emails_input.invalid_email'
+            defaultMessage='Type a valid email address to invite new people'
```
Comment (grundleborg) [inline components/widgets/inputs/emails_input.jsx]: Same here.

File: components/widgets/inputs/emails_input.jsx (hunk 1 above)
Comment (jespino) [reply components/widgets/inputs/emails_input.jsx]: Fixed

File: components/widgets/inputs/emails_input.jsx (hunk 2 above)
Comment (jespino) [reply components/widgets/inputs/emails_input.jsx]: Fixed

**Pull request merged**